rails/app/controllers/super_admin/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+module SuperAdmin
+  class UsersController < ApplicationController
+    def edit
+      @user = User.find(params[:id])
+
+      if @user.role == 'admin'
+        @user
+      else
+        flash[:alert] = 'You are not authorized to perform this action.'
+        redirect_to(root_path)
```

Comment:
I'm wondering if it would be better for us to redirect to the community path rather than to the root path which shows the metrics for all of the communities for the super admin. Since it seems the current path is: Metrics root page (after login) -> Communities -> Community -> Find the user in the tiny list -> edit. On save, going all the way back feels jarring.

Alternatively, it may also be worth creating a new list view under the Super Admin dashboard that lists all of the admin users in the system that we can use to redirect to and make it easier for super admins to reset admin passwords, regardless of the community.

@rudokemper thoughts?

Reply:
Sorry I missed this back in May. I think the first solution makes the most sense.